src/zope/schema/_bootstrapinterfaces.py

```diff
@@ -230,10 +278,23 @@ def __init__(self, errors=None, name=None, schema_errors=None, invariant_errors=
         self.schema_errors = schema_errors
         self.invariant_errors = invariant_errors
 
+    def _describe(self):
+        error_strs = []
+        if self.schema_errors:
+            error_strs.extend([
+                '%s: %s' % (name, error)
+                for name, error in sorted(self.schema_errors.items())])
+        if self.invariant_errors:
+            error_strs.extend([str(error) for error in self.invariant_errors])
+        return '\n'.join(error_strs)
```

Comment:
I worry that embedded newlines inside each field's error will make this harder to read.

I would be tempted to fix that by doing `'%s: %s' % (name, str(error).replace('\n', '\n  '))` to preserve the hierarchy with indentation.  That should suffice to make it clearer which error refers to which field in text logs even if you have multiple levels of nesting.